tests/BUILD

```diff
@@ -90,3 +91,13 @@ cc_library(
         "@unity",
     ],
 )
+
+cc_test(
```

Comment:
test case will be done in test serialization. So we don't have to do this for now either.